proto/cosmos/types.proto

```diff
@@ -1,8 +1,8 @@
 syntax = "proto3";
-package cosmos_sdk.v1;
+package cosmos;
```

Comment:
An alternative would be `cosmos.base`

Reply:
Sounds good to me. Will wait for a few other opinions, and then i can go ahead & make these changes.

Reply:
Maybe we make this `cosmos.base` but keep `cosmos.crypto` and `cosmos.tx` as is? Not sure...